setup.cfg

```diff
@@ -11,6 +11,8 @@ addopts = --doctest-modules --cov=./src
 
 [flake8]
 exclude = __init__.py,versioneer.py,src/_version.py
+per-file-ignores =
+    tests/*: D
```

Comment:
Perchè non metterlo direttamente negli `exclude`? flake è davvero così figo da ignorare per-file solo alcuni specifici controlli (in questo caso quelli legati alla documentazione)? Se è così, chapeau!

In ogni caso però dall'exclude andrebbe rimosso `__init__.py` che in effetti non esiste.

Reply:
Eh ma se metti exclude esclude il controllo dei codici relativi alle doc strings anche dal pacchetto. Cmq si flake8 è davvero così figo...

Reply:
Ah l'ho testato nell'altra libreria di prova, quindi si, funziona davvero

Reply:
Ottimo!